opendevin/core/logger.py

```diff
@@ -161,14 +160,18 @@ def log_uncaught_exceptions(ex_cls, ex, tb):
 opendevin_logger.setLevel(logging.INFO)
 if DEBUG:
     opendevin_logger.setLevel(logging.DEBUG)
-    opendevin_logger.addHandler(get_file_handler())
+    # default log to project root
+    log_dir = os.path.join(
+        os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))),
```

Comment:
hmm is this one too many?

Reply:
The idea was to save on the project root, not inside the `opendevin`, so three folders up should be right